docs/application/web/api/6.5/device_api/wearable/tizen/ml.html

```diff
@@ -1583,9 +1594,10 @@ <h2 id="full-webidl">4. Full WebIDL</h2>
 <pre class="webidl prettyprint">module MachineLearning {
   typedef (byte[] or TypedArray) Bytes;
   enum TensorType { "INT8", "UINT8", "INT16", "UINT16", "FLOAT32", "INT32", "UINT32", "FLOAT64", "INT64", "UINT64", "UNKNOWN" };
-  enum HWType  { "ANY", "AUTO", "CPU", "CPU_NEON", "CPU_SIMD", "GPU", "NPU", "NPU_EDGE_TPU", "NPU_MOVIDIUS", "NPU_SR", "NPU_VIVANTE" };
-  enum NNFWType { "ANY", "ARM_NN", "CUSTOM_FILTER", "EDGE_TPU", "MVNC", "NNFW", "OPEN_VINO", "SNPE", "TENSORFLOW", "TENSORFLOW_LITE",
-    "VIVANTE" };
+  enum HWType  { "ANY", "AUTO", "CPU", "CPU_NEON", "CPU_SIMD", "GPU", "NPU", "NPU_EDGE_TPU", "NPU_MOVIDIUS", "NPU_SLSI", "NPU_SR",
+    "NPU_VIVANTE" };
+  enum NNFWType { "ANY", "ARM_NN", "CUSTOM_FILTER", "EDGE_TPU", "MVNC", "NNFW", "NNTR_INF", "OPEN_VINO", "PYTORCH", "SNPE", "SRNPU",
+    "TENSORFLOW", "TENSORFLOW_LITE", "VD_AIFW", "VIVANTE" };
```

Comment:
I reviewed this file again to check for any broken links in the document. I didn't find any broken links in this file.